pkg/reconciler/v1alpha1/configuration/resources/revision_test.go

```diff
@@ -175,8 +175,8 @@ func TestMakeRevisions(t *testing.T) {
 					serving.ConfigurationGenerationLabelKey:                   "100",
 					serving.DeprecatedConfigurationMetadataGenerationLabelKey: "100",
 					serving.ServiceLabelKey:                                   "",
-					"foo":                                                     "bar",
-					"baz":                                                     "blah",
+					"foo": "bar",
+					"baz": "blah",
```

Comment:
Is this diff wanted?

Reply:
This is automatically changed by my IDE, Visual Studio Code. I guess this is guided by golang lint?

Reply:
I'd say it shouldn't update this. Anyway 🤷‍♂️